api/api-doc/storage_service.json

```diff
@@ -2110,6 +2110,65 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/tombstone_gc{keyspace}",
```

Comment:
missing `/` in path, before `{keyspace}`

Reply:
Does it have a rest api test?

Reply:
it does

Reply:
Hmm, I wonder how it passed it then

Reply:
So do I. Will check